src/index-node.ts

```diff
@@ -24,4 +25,13 @@ export class SiweClient extends SiweImpl {
   ) {
     super(config, signingFunction, fetchWithCookie)
   }
+
+  async _extractCookies(_header?: Headers): Promise<void> {
+    const cookieRecord: Record<string, string> = {}
+    const setCookie = Cookie.parse(_header?.get('set-cookie') || '')
```

Comment:
```suggestion
  async _extractCookies(headers?: Headers): Promise<void> {
    const cookieRecord: Record<string, string> = {}
    const setCookie = Cookie.parse(headers?.get('set-cookie') || '')
```
you don't need `_` here since this function uses the param